modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -177,8 +179,8 @@ You can use these rules to define an API version:
 == Document an API Version
 
 Exchange provides documentation pages with information about an asset for API consumers. 
-You can create different documentation pages for each API Version.
+You can create different documentation pages for each minor asset version.
 
-The API version that is being edited can be changed from the button next to the API Name.
+The minor asset version that is being edited can be changed with the menu next to the API name.
```

Comment:
Intentional passive voice?

Reply:
No, I just didn't do enough rewriting from the existing version. Fixed. Thanks!